src/Package/Impl/Utilities/Navigation.cs

```diff
@@ -21,7 +22,7 @@ public static class Navigation {
         /// <summary>
         /// Activates a text view for a text buffer, and sets the cursor to a specific location
         /// </summary>
-        public static bool NavigateToTextBuffer(ITextBuffer textBuffer, int start, int length) {
+        public static bool NavigateToTextBuffer(IServiceContainer services, ITextBuffer textBuffer, int start, int length) {
```

Comment:
I was planning to turn this into some 'navigation service' but it's OK to keep it static for now